cmd/miner/client.go

```diff
@@ -105,17 +104,19 @@ func (m *Miner) subscribe() error {
 	return nil
 }
 
+// sleep pauses goroutine execution for the provided duration.
+func (m *Miner) sleep(duration time.Duration) {
```

Comment:
It doesn't really matter much, but I don't see why this is needed.  Why not just call `time.Sleep` directly?

Reply:
updated.